motors/tacho_motor_class.c

```diff
@@ -255,7 +251,7 @@ struct tacho_motor_state_item {
 
 static struct tacho_motor_value_names tacho_motor_states[NUM_TM_STATES] = {
 	[TM_STATE_RUNNING]	= { "running" },
-	[TM_STATE_RAMPING]	= { "ramping" },
+	[TM_STATE_STOPPED]	= { "stopped" },
```

Comment:
Why did you remove the ramping flag? I think it is useful to know if the motor speed is being regulated by ramping or not.

Also, these values are flags, so you can have more than one at a time. I'm not sure it makes sense to have a `stopped` flag unless it means something that the presence or absence of other flags does not already indicate.